apis/python/src/tiledbvcf/__init__.py

```diff
@@ -3,13 +3,16 @@
 
 import ctypes
 import os
+import platform
```

Comment:
import platform not needed if other changes to sys.platform made

Reply:
It looks like the two approaches (`sys.platform` versus `platform.system`) are equivalent

https://docs.python.org/3/library/sys.html#sys.platform
https://docs.python.org/3/library/platform.html#platform.system

But since the existing code uses `sys.platform` in the main codebase (it uses `platform.system` in the tests), I'll switch to it for consistency

https://github.com/TileDB-Inc/TileDB-VCF/blob/a5589734a5eb4e1cae5037e4b6b18f02d6d58efd/apis/python/src/tiledbvcf/__init__.py#L11